src/components/Buttons/GoBackButton/GoBackButton.scss

```diff
@@ -0,0 +1,13 @@
+@use "../../../styles/partials/mixins" as *;
+@use "../../../styles/partials/variables" as *;
+
+.go-back-button {
+    background-color: $primary-white;
+    border: none;
+    align-self: center;
+    padding-top: .6rem;
+
```

Comment:
The button isn't centred because f width and height. You don't need to use align-self and padding-top. Just assign   ```
width: 1.5rem;
height: 1.5rem;, 
```
it will work